modules/community.py

```diff
@@ -0,0 +1,41 @@
+""" Module for interactions with everyone.
+
+A command belongs here if it involves more than one person.
+
+A command does *not* belong here if it...
+ - operates without any call or response, or
+ - is an introvert.
+"""
+
+import emoji
+import re
+from modules import module
+from modules import shared
+
+
+_server_emoji_re = re.compile(r'<:.*:(.*)>')
+
+
+def register_commands():
+    @module.module_command
+    async def poll(message):
+        """```poll <question>
+        <answer 1> <emojus>
+        ...
+        <answer N> <emojus>```
+        Shortcut to add emoji as reactions to a poll question.
+
+        Note: The bot is, sadly, unable to handle emoji from other servers.
```

Comment:
I'm not sure this is entirely true; we have an emote server for Enigma that has the Septapus bot in it, and it's able to grab emotes from other servers to present a full-size version via a `hugemoji` command. Don't know if it requires something silly like giving the bot Nitro, though. Worth further investigation.